primefaces/src/main/java/org/primefaces/component/dialog/DialogRenderer.java

```diff
@@ -116,6 +116,9 @@ protected void encodeMarkup(FacesContext context, Dialog dialog) throws IOExcept
             writer.writeAttribute("style", style, null);
         }
 
+        writer.startElement("div", null);
+        writer.writeAttribute("class", Dialog.CONTAINER_CLASS, null);
```

Comment:
Looks like ConfirmDialog selenium tests are failing it might be related to this new element.